services/wallet/commands.go

```diff
@@ -292,6 +293,10 @@ func (c *newBlocksTransfersCommand) checkDelay(parent context.Context, nextHeade
 }
 
 func (c *newBlocksTransfersCommand) Run(parent context.Context) (err error) {
+	if !c.service.IsAppActive() {
```

Comment:
Will `c.from` be the same on the next invocation?

Reply:
yes sure, it is like last known block here. The idea is that we do not touch the way how this command recovers after the long delay before fetching next block, we just "create" this delay